Gopkg.lock

```diff
@@ -1079,6 +1079,14 @@
   pruneopts = "UT"
   revision = "3a2206dd6a78497deceb3ae058417fdeb2036c7e"
 
+[[projects]]
```

Comment:
Why `Gopkg.lock` was updated but `Gopkg.toml` was not? I think we do not need to update `Gopkg.lock` but it should be updated by `dep ensure`?

Also for such PR, it is better have two commits for better review. One for you manually updated code, and another for auto-generated code via `dep ensure`.

Reply:
`Gopkg.lock` was updated because it's a new implicit dependency pulled in by import rather than being explicitly added as a constraint in `Gopkg.toml` I would suspect that running `dep ensure` is what updated `Gopkg.lock` here.

I do agree that in general that having dependency updates in a separate commit helps reviews, but in this case the diff is relatively small, so I don't consider it blocking review here.